app/src/main/res/values/strings.xml

```diff
@@ -93,4 +94,57 @@
     <string name="firstrun_description">Automatically block online trackers while you browse. Then tap to erase visited pages, searches, cookies and passwords from your device.</string>
     <string name="firstrun_button">Ok, got it!</string>
     <string name="firstrun_footer">To be clear, sites and service you use can still know where you\'ve been.</string>
+
+    <!-- The page html title (i.e. the <title> tag content) -->
+    <string name="errorpage_title">Problem loading page</string>
+    <string name="errorpage_refresh">Try Again</string>
+
+    <string name="error_connectionfailure_title">Unable to connect</string>
+    <string name="error_connectionfailure_message">
+    <![CDATA[
+      <ul>
+        <li>The site could be temporarily unavailable or too busy. Try again in a few moments.</li>
+        <li>If you are unable to load any pages, check your mobile device’s data or Wi-Fi connection.</li>
+      </ul>
+    ]]>
```

Comment:
We need to test whether this works with our translation export/import. If not - maybe we could split that into separate strings without HTML?